src/commons/Constants.ts

```diff
@@ -82,6 +82,12 @@ function setAccessibility() {
 
 setAccessibility();
 
+const WebDeviceFrameSelector = '[data-react-native-web-dimensions="true"]';
```

Comment:
Is the selector is something you have in the editor or is it a general selector that web apps have? 

How does it work with both stage and settings panel?

Reply:
Yes, it's a check we added to a div wrapping the stage 'device'.
In settings we don't have this problem since the UI is less complex and not involved (as of today) things related to screen size.
Since we don't have a context or something that wraps uilib context, we don't "know" where this will be called from - it's one single uilib for all usages

Reply:
Let's move the code to private uilib's 
We have Constants overrides